src/core/heating_cooling_loads.jl

```diff
@@ -9,11 +9,13 @@
     industrial_reference_name::String = "",  # For ProcessHeatLoad
     blended_industrial_reference_names::Array{String, 1} = String[],  # For ProcessHeatLoad
     blended_industrial_reference_percents::Array{<:Real,1} = Real[],  # For ProcessHeatLoad
-    addressable_load_fraction::Any = 1.0,  # Fraction of input fuel load which is addressable by heating technologies. Can be a scalar or vector with length aligned with use of monthly_mmbtu or fuel_loads_mmbtu_per_hour.
+    city::String = "",
+    year::Union{Int, Nothing} = doe_reference_name ≠ "" || blended_doe_reference_names ≠ String[] ? 2017 : nothing, # CRB profiles are 2017 by default. If providing load profile, specify year of data.    
```

Comment:
Does this year check for defaulting to 2017 also need to account for `industrial_reference_name` and `blended_industrial_reference_names`? I'm not familiar with exactly how those are used.

Reply:
Good catch, I'll add that too.